mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js

```diff
@@ -301,4 +303,11 @@ const styles = StyleSheet.create({
         width: "100%",
         marginBottom: verticalScale(18),
     },
+    answerText: {
+      marginLeft: moderateScale(25),
+      marginRight: moderateScale(25),
+      marginHorizontal: scale(20),
+      marginVertical: scale(20),
```

Comment:
For vertical, we use `verticalScale`

Reply:
resolved through: https://github.com/rightoneducation/righton-app/pull/518